src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -95,9 +96,99 @@ public int getBallCount() {
   public void expelBalls() {
     if (ballCount > 2) {
```

Comment:
I think you want to call getBallCount() here instead of using the ballCount variable. Currently getBallCount() never gets run anywhere so ballCount will aways be 0.

Reply:
Skipping this for now in the PR